app/models/partnership.rb

```diff
@@ -0,0 +1,13 @@
+class Partnership < ApplicationRecord
+  belongs_to :school
+  belongs_to :lead_provider
+
+  def confirmed?
+    !confirmed.nil?
```

Comment:
Could be just me but I find double negations hard to understand, why not just confirmed.present?